### ISSUE TITLE
[FEATURE] Bump monitoring chart version

### ISSUE THREAD

**Issue by guangbochen** [harvester/harvester#1787]
**Is your feature request related to a problem? Please describe.**
Bump monitoring chart version to sync with the latest rancher v2.6 release. 

Refer to: https://github.com/rancher/charts/tree/dev-v2.6/charts/rancher-monitoring

**Comment by guangbochen** [comment]
To Test:
- verify the air-gap env works with the latest monitoring chart
- verify both the cluster and VM monitoring metrics are showing properly on both the home page and the VM metrics page.

**Comment by TachunLin** [comment]
Verified **fixed** on master-a67f96d0-head (1/26). Close this issue.

### Result
`Pass` the following check points on 3 nodes harvester under air-gap environments
 
1. Monitoring chart bump to `100.1.0+up19.0.3` compare with 
    https://github.com/rancher/charts/tree/dev-v2.6/charts/rancher-monitoring
  ```
  harvester-node-0:~ # yq e '.monitoringChart' /etc/harvester-release.yaml
  100.1.0+up19.0.3
  ```
2. Host cluster monitoring chart display properly on home pahge
  ![image](https://user-images.githubusercontent.com/29251855/151123556-36183a4d-37ce-4255-93fd-10db7e9a53e6.png)

3. VM monitoring metrics display properly on VM metrics page
  ![image](https://user-images.githubusercontent.com/29251855/151126785-c0692259-2bbf-4fb4-8d8e-2197f313cff8.png)
  ![image](https://user-images.githubusercontent.com/29251855/151126814-4ccbf15e-054b-4c33-9285-e44a954ab134.png)


### Test Information
* Test Environment: 3 nodes harvester on local kvm machines
* Harvester version: master-a67f96d0-head (1/26)

### Verify Steps
1. User ipx vagrant install 3 nodes harvester (with `offline` enabled) 
2. Upload images from file
3. Enable virtual network with harvester mgmt 
4. Create virtual network `vlan1` with id `1`
5. Create a virtual machine 
6. Check monitoring chart status on harvester home page
7. Wait for VM in running status
8. Check monitoring metrics in VM metrics page
